source/reference/minio-server/settings/notifications/mqtt.rst

```diff
@@ -36,11 +36,11 @@ For example, the following commands set two distinct MQTT service endpoints as `
       .. code-block:: shell
```

Comment:
Is there a `windows` | `not windows` differentiator for this text?

Reply:
no, we don't produce windows docs for the Minio Server reference docs. They're linux only.